Database/GameDB/src/main/java/com/csc480red/gamedb/Score.java

```diff
@@ -0,0 +1,187 @@
+package com.csc480red.scoresdb;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+public class Score {
```

Comment:
Please abstract this into some sort of Tile class where each Tile has a letter and a Score, and load all the tiles from the database from a letter table (i.e TileTable)

Reply:
such as this.
```java
public class Tile implements Serializable {
	
	private final char letter;
	private final int value;

	public class Tile(char letter, int value) {
		this.letter = letter;
		this.value = value;
	}

	public char getLetter() {
		return letter;
	}

	public int getValue() {
		return value;
	}
}
```

Reply:
Another way of looking at it is with a HashMap like:
```java
public class Score {
	
	private Map<String, Integer> data;

	public Score() {
		data = new HashMap<>();
	}

	public Score(Map<String, Integer> data) {
		this.data = data;
	}

	public void add(String letter, Byte value) {
		data.put(letter, value);
	}

	public HashMap<String, Byte> getData() {
		return data;
	}
}
```